protos/local/temporal/sdk/core/workflow_activation/workflow_activation.proto

```diff
@@ -186,25 +186,25 @@ message QueryWorkflow {
     string query_id = 1;
     /// The query's function/method/etc name
     string query_type = 2;
-    repeated common.Payload arguments = 3;
+    repeated temporal.api.common.v1.Payload arguments = 3;
```

Comment:
No real benefit, but could use `temporal.api.common.v1.Payloads` instead of `repeated` if you wanted